### PR TITLE
Update italian translations for solidus 2.9

### PR DESCRIPTION
**Pull request by delphaber**
I started from https://github.com/solidusio/solidus_i18n/pull/138 ( thanks @afdev82 ! ) and I used the `bundle exec i18n-tasks add-missing --nil-value --locale it` task to update the translations for solidus 2.9

I also proofread most of translations.

The PR is still in WIP because I think I will find more translations to fix in the next days.

### CODE REVIEW

File: config/locales/it.yml [hunk 1]
```diff
@@ -988,18 +1522,18 @@ it:
...
+    listing_reports: Elenco Reports
+    listing_tax_categories: Elenco Categorie di Tassazione
+    listing_users: Elenco Utenti
     loading: Caricamento
     locale_changed: Lingua cambiata
     location: Luogo
     lock: Blocca
     log_entries: Voci Log
-    logged_in_as: Login come
+    logged_in_as: Loggato come
```
Comment (spaghetticode) [inline config/locales/it.yml]: Maybe `Autenticato come` would sound better?

File: config/locales/it.yml [hunk 2]
```diff
@@ -1035,10 +1573,12 @@ it:
     name: Nome
     name_on_card: Nome sulla carta
     name_or_sku: Nome o SKU (inserisci almeno i primi 4 caratteri del nome del prodotto)
+    negative_movement_absent_item: Impossibile creare movimento negativo per articoli
```
Comment (spaghetticode) [inline config/locales/it.yml]: what about `per articoli assenti in magazzino`?

File: config/locales/it.yml [hunk 3]
```diff
@@ -1050,50 +1590,65 @@ it:
...
-    no_resource_found_link: Aggiungine Uno
+    no_promotions_found: Nessuna promozione trovata
+    no_resource: "Nessuna risorsa di tipo '%{resource}' trovata."
+    no_resource_found: "Nessuna risorsa di tipo '%{resource}' trovata."
+    no_resource_found_html: Nessuna risorsa di tipo '%{resource}' trovata, %{add_one_link}!
+    no_resource_found_link: Aggiungine uno/a
     no_results: Nessun risultato
-    no_returns_found: Nessuna restituazione trovata
-    no_rules_added: Nessuna regola aggiunta
+    no_rules_added: Nessa regola aggiunta
```
Comment (spaghetticode) [inline config/locales/it.yml]: `Nessa` -> `Nessuna`

File: config/locales/it.yml [hunk 4]
```diff
@@ -1050,50 +1590,65 @@ it:
...
     no_shipping_method_selected: Nessun metodo di spedizione selezionato.
-    no_state_changes: Nessun cambio di stato
-    no_tracking_present: Non sono stati forniti dettagli sul tracciamento
+    no_shipping_methods_found: Nessun metodo di spedizione trovato
+    no_stock_locations_found: Nessun magazzino trovato
+    no_trackers_found: Nessun tracker trovato
+    no_tracking_present: Non sono stati forniti dettagli sul tracciamento.
+    no_variants_found: Nessuna variante trovata.
+    no_variants_found_try_again: Nessuna variante trovata. Prova a cambiare la tua ricerca.
```
Comment (spaghetticode) [inline config/locales/it.yml]: what about `cambiare` -> `modificare` ?

**Comment by delphaber** [comment]
Thank you for the review @spaghetticode ! I'm gonna apply your suggestions :)

File: config/locales/it.yml (hunk 1 above)
Comment (delphaber) [reply config/locales/it.yml]: done! thanks!

File: config/locales/it.yml (hunk 2 above)
Comment (delphaber) [reply config/locales/it.yml]: you are right! :) thank you. done!

File: config/locales/it.yml (hunk 3 above)
Comment (delphaber) [reply config/locales/it.yml]: typo! :D thx!

File: config/locales/it.yml (hunk 4 above)
Comment (delphaber) [reply config/locales/it.yml]: sure thing :) thank you!

**Comment by delphaber** [comment]
I will squash all of these commits and I'll wait for https://github.com/solidusio/solidus_i18n/pull/163/files to be merged, because I will change `js_format` too in my PR

**Comment by delphaber** [comment]
@spaghetticode I rebased on top of master and squashed all the commits into only 1 :)

I think it's ready to be merged!

**Comment by delphaber** [comment]
@spaghetticode can this be merged? Any concern? Thanks!

**Pull request merged**